humble-harmonic/Dockerfile

```
@@ -0,0 +1,25 @@
+####################################
+# Ubuntu ROS2 Humble Gazebo Fortress
+####################################
+
+FROM ghcr.io/soham2560/humble:latest AS base
+
+# Environment setup
+ENV GZ_VERSION=harmonic
+
+# install necessary tools
+RUN sudo apt-get update && sudo apt-get install --no-install-recommends -y \
+    curl \
+    lsb-release \
+    wget \
+    gnupg \
+    && sudo rm -rf /var/lib/apt/lists/*
+
+# Install gazebo fortress from osrf
+RUN sudo curl https://packages.osrfoundation.org/gazebo.gpg --output /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null \
+    && sudo apt-get update \
+    && sudo apt-get install -y gz-harmonic \
```

any reason to move away from the standard args for install?

https://github.com/soham2560/DockerForROS2Development/blob/2420e6c643a3073325b021115f4c407ae29482f4/humble-garden/Dockerfile#L20-L22

```suggestion
    && sudo apt-get -qq update \
    && sudo xargs apt-get install --no-install-recommends -y \
    gz-harmonic \
```

---

Thanks for noticing it, it was not intentional, I have updated refer [5a85ecf](https://github.com/soham2560/DockerForROS2Development/pull/19/commits/5a85ecf7b1e80ae8b91d7a7262161e89c126422e)